README.md

```diff
@@ -5,7 +5,7 @@
     <img alt="CEH" src="https://img.shields.io/badge/policy-Cardano%20Engineering%20Handbook-informational?style=for-the-badge" />
   </a>
   <a href="https://github.com/input-output-hk/cardano-ledger/actions/workflows/haskell.yml">
-    <img alt="GitHub Workflow Status (master)" src="https://img.shields.io/github/workflow/status/input-output-hk/cardano-ledger/Haskell%20CI/master.svg?style=for-the-badge" />
+    <img alt="GitHub Workflow Status (master)" src="https://img.shields.io/github/actions/workflow/status/input-output-hk/cardano-ledger/workflows/haskell.yml?branch=master&style=for-the-badge" />
```

Comment:
```suggestion
    <img alt="GitHub Workflow Status (master)" src="https://img.shields.io/github/actions/workflow/status/input-output-hk/cardano-ledger/haskell.yml?branch=master&style=for-the-badge" />
```

Reply:
I tried it without it originally, and it did not work. The issue reads:

> If your workflow is in a subdirectory under .github, include the subdir in the route.
>
> `https://img.shields.io/github/actions/workflow/status/<user>/<repo>/sub_directory/test.yml?branch=main`

and our `haskell.yml` file is located in `.github/workflows/`.

Reply:
nm, that works :shrug: 

thanks!

Reply:
Checking badges is pretty easy, you can always open the link in the browser and see how it looks like :wink: 

![Badge](https://img.shields.io/github/actions/workflow/status/input-output-hk/cardano-ledger/haskell.yml?branch=master&style=for-the-badge)